docs/getting-started/tutorials/local_setup.md

```diff
@@ -51,8 +51,8 @@ Java HotSpot(TM) 64-Bit Server VM (build 25.111-b14, mixed mode)
 
 <div data-lang="Download and Unpack" markdown="1">
 1. Download a binary from the [downloads page](http://flink.apache.org/downloads.html). You can pick
-   any Hadoop/Scala combination you like. If you plan to just use the local file system, any Hadoop
-   version will work fine.
+   any Scala variant you like. For certain features you may also have to download one of the pre-bundled hadoop jars
+   and place them into the `/lib` directory.
```

Comment:
where do the users get `the pre-bundled hadoop jars`? I guess the page will be read by people very new to Flink.
 is it now not only for usage with the local file system or hadoop locally? it could be still mentioned in brackets:
`you may also have ... for certain features (to use the local file system, ... etc)`

Reply:
On the download page that was just linked; it's listed in the optional components section for a release. It should be easy enough to find.

Reply:
True, have not noticed it and I guess who needs hadoop knows better for which features they need it.